src/plugins/data_view_field_editor/public/open_editor.tsx

```diff
@@ -153,7 +153,11 @@ export const getFieldEditorOpener =
 
       let field: Field | undefined;
       if (dataViewField) {
-        if (isExistingRuntimeField && dataViewField.runtimeField!.type === 'composite') {
+        if (
```

Comment:
When should we end up in here? In my testing, it didn't seem like these conditions were met for either composite fields or their subfields.

Reply:
Good catch! You're right, that code wasn't being executed. Its removed.